tests/orquestra/quantum/circuits/_circuit_test.py

```diff
@@ -5,7 +5,13 @@
 import pytest
 import sympy
 
-from orquestra.quantum.circuits import GateOperation, MultiPhaseOperation, split_circuit
+from orquestra.quantum.circuits import (
+    GateOperation,
+    MultiPhaseOperation,
+    split_circuit,
+    CustomGateDefinition,
+    Circuit
```

Comment:
Black is complaining about this:
```suggestion
    Circuit,
```